modules/nf-core/pymygene/tests/main.nf.test

```diff
@@ -0,0 +1,43 @@
+nextflow_process {
+
+    name "Test Process PYMYGENE"
+    script "../main.nf"
+    process "PYMYGENE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pymygene"
+
+    test("pymygene - txt") {
+        when {
+            process {
+                """
+                // Create a temporary file with the first column of the input file, without header
+                source = params.test_data['mus_musculus']['genome']['rnaseq_matrix']
```

Comment:
Not very well documented yet, but can you use a construct like [this one](https://github.com/nf-core/modules/blob/cabcc0dadf8366aa7a9930066a7b3dd90d9825d5/modules/nf-core/samtools/index/tests/main.nf.test#L21) for the test data, instead of the test_data map? That will make the tests portable to the workflow level (when we enable nf-test in differentialabundance!)